src/Tribe/Tickets_Handler.php

```diff
@@ -531,22 +365,25 @@ public function get_event_shared_tickets( $post = null ) {
 		$tickets     = Tribe__Tickets__Tickets::get_event_tickets( $post_id );
 		$ticket_list = array();
 
-		if ( ! empty( $tickets ) ) {
-			foreach ( $tickets as $ticket ) {
-				$stock_mode = $ticket->global_stock_mode();
-				if ( empty( $stock_mode ) || Tribe__Tickets__Global_Stock::OWN_STOCK_MODE == $stock_mode ) {
-					continue;
-				}
+		if ( empty( $tickets ) ) {
+			return $ticket_list;
+		}
 
-				// Failsafe - should not include unlimited tickets
-				if ( $this->is_unlimited_ticket( $ticket ) ) {
-					continue;
-				}
+		foreach ( $tickets as $ticket ) {
+			$stock_mode = $ticket->global_stock_mode();
+			if ( empty( $stock_mode ) || Tribe__Tickets__Global_Stock::OWN_STOCK_MODE == $stock_mode ) {
```

Comment:
Yodify second part of condition

Reply:
That is the YODA version. You cannot assign values to Constants.